### ISSUE TITLE
traci.load in sumo gui may use the scale of the last sumocfg instead of the new one

### ISSUE THREAD

**Issue by 1952890** [eclipse/sumo#13627]
traci.load will use the scale of the last sumocfg instead of the new one 

examples
cmd2 = [sumoBinary,'-c',cfg_path]
traci.start(cmd2)
traci.load(['-c',sumocfg_traciload])

if “cfg_path” and “sumocfg_traciload” do not have the same scale of map , the new map will use the scale of “cfg_path” instead of the one from “sumocfg_traciload” 
By the way, the center of the new map could not be placed in the center of the window

![image](https://github.com/eclipse/sumo/assets/110601017/2145e02d-d752-40f7-bd9c-9a26aefcf237)

**SUMO-version:1.18**

**operating system:windows**

**Comment by m-kro** [comment]
You are right that calling `traci.load` will not recenter the network as would do `File > Open Simulation...` in sumo-gui. We could change that...

**Comment by m-kro** [comment]
@1952890 As we will not change this shortly, I recommend you use other TraCI calls to center the network in the GUI in case you need an automated approach. Have a look into [traci.simulation.getNetBoundary](https://sumo.dlr.de/pydoc/traci._simulation.html#SimulationDomain-getNetBoundary) & [traci.gui.setBoundary](https://sumo.dlr.de/pydoc/traci._gui.html#GuiDomain-setBoundary).
